src/content/docs/apis/synthetics-rest-api/monitor-examples/manage-synthetics-monitors-rest-api.mdx

```diff
@@ -21,7 +21,11 @@ Use the synthetics REST API to create and manage [synthetic monitors](/docs/synt
 ## Before you start [#before-you-start]
 
 <Callout variant="important">
-  On October 22, 2024, we will [end of life](/whats-new/2024/04/whats-new-04-09-eol-synthetics-runtime-cpm) the containerized private minion (CPM) and legacy synthetics runtime versions. On June 30, 2024, we will block the creation of new monitors using legacy synthetics runtime versions. The Synthetics REST API only supports monitor creation using legacy synthetics runtime versions. Please use [NerdGraph APIs](/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial/) to manage your synthetic monitors using our latest runtimes to avoid degradation.
+  The Synthetics REST API only supports monitor creation using legacy synthetics runtime versions. Please use [NerdGraph APIs](/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial/) to manage your synthetic monitors using our latest runtimes.
+</Callout>
+
+<Callout variant="important">
+  On October 22, 2024, we will [end of life](/whats-new/2024/04/whats-new-04-09-eol-synthetics-runtime-cpm) the containerized private minion (CPM) and legacy synthetics runtime versions. On June 30, 2024, we will block the creation of new monitors using legacy synthetics runtime versions. 
```

Comment:
On October 22, 2024, the containerized private minion (CPM) and legacy synthetics runtime versions reach [end of life](/whats-new/2024/04/whats-new-04-09-eol-synthetics-runtime-cpm). From June 30, 2024, we have blocked the creation of new monitors using legacy synthetics runtime versions. (since june 30 has passed)